languages/julia/exercises/concept/poetry-club/.docs/instructions.md

```diff
@@ -0,0 +1,55 @@
+A new poetry club has opened in town, and you're thinking of attending.
+Because there have been incidents in the past, the club has a very specific door policy which you'll need to master, before attempting entry.
+
+There are two doors at the poetry club, both are guarded.
+In order to gain entry, you'll need to work out the password of that day:
+
+### Front door
+
+1. The guard will recite a poem, one line at the time;
+   - You will have to respond with the appropriate letter.
+2. The guard will tell you all the letters you've responded with at once;
+   - You need to format the letters as a capitalised word.
```

Comment:
This bit doesn't seem to correspond to the tests (instead, we're told the entire poem and have to give the password).

It also doesn't make a lot of sense to talk about a word being capitalised when the scenario is verbal.

Reply:
I forgot to change that from the JS implementation where the input for the password is an array. I think giving the whole poem makes more sense, what do you think?

> It also doesn't make a lot of sense to talk about a word being capitalised when the scenario is verbal.

Later in the text it's clarified that you're meant to write down that password. I'll clarify this.

Reply:
I think we should give them the entire poem and have them split it.

Reply:
The reason I don't have them split it is because this comes _before_ array in my tracks. I think.